Sources/Pageboy/PageboyViewController.swift

```diff
@@ -20,7 +20,7 @@ open class PageboyViewController: UIViewController {
 
     // MARK: Properties
     
-    internal var pageViewController: UIPageViewController?
+    internal (set) public var pageViewController: UIPageViewController?
```

Comment:
I'd prefer to keep the internal `UIPageViewController ` private API so that accidents can't happen relating to the `dataSource` / `delegate` and other things Pageboy manipulates. Is there any reason this needs to be `public`?